internal/plugin.go

```diff
@@ -648,6 +651,31 @@ func (p *Plugin) assertValidConfig() error {
 		)
 	}
 
+	// validate placement and binding names are DNS compliant
+	placementName := ""
+	if p.PolicyDefaults.Placement.PlacementRuleName != "" {
```

Comment:
You'll also need to duplicate this logic for each policy since these values could have been overwritten.